applications/vanilla/models/class.discussionmodel.php

```diff
@@ -289,10 +292,9 @@ public function get($Offset = '0', $Limit = '', $Wheres = '', $AdditionalFields
         }
 
         // Get preferred sort order
-        $SortField = self::GetSortField();
+        $orderFields = $this->getOrderFields();
 
-        $this->EventArguments['SortField'] = &$SortField;
-        $this->EventArguments['SortDirection'] = c('Vanilla.Discussions.SortDirection', 'desc');
+        $this->EventArguments['OrderFields'] = &$orderFields;
```

Comment:
This pull request is to support multiple order by clauses. Keeping the single string-type sort field as an argument means that we would need to build extra logic (i.e., if the OrderFields exist, use that, otherwise default to the SortField... etc.) Consolidating the sort fields and directions into one array is simpler and more explicit in that we know which direction belongs to which field.

I've done a search to see whether these arguments are even being used in Vanilla and it came up empty. I think it's a pretty safe enhancement.